tests/test_app_cli.py

```diff
@@ -138,6 +139,21 @@ async def test_export_to_json(tmpdir, add_types):
 
     os.chdir(workspace)
 
+    install_args = [
+        tmpvenv_active_inherit.python_path,
+        "-m",
+        "inmanta.app",
+        "project",
+        "install",
+    ]
+    install_process = await subprocess.create_subprocess_exec(*install_args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    try:
+        (stdout, stderr) = await asyncio.wait_for(install_process.communicate(), timeout=30)
+    except asyncio.TimeoutError as e:
+        install_process.kill()
+        await install_process.communicate()
+        raise e
+
```

Comment:
Shouldn't we isolate this logic into a separate method.

Reply:
Good suggestion, done.